Sources/_OpenAPIGeneratorCore/PlatformChecks.swift

```diff
@@ -0,0 +1,18 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftOpenAPIGenerator open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftOpenAPIGenerator project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftOpenAPIGenerator project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+// Details: https://github.com/apple/swift-openapi-generator/issues/86
+#if os(iOS) || os(tvOS) || os(watchOS)
```

Comment:
I think this is what we want. Especially with upcoming `visionOS`.

```suggestion
#if !os(macOS) || !os(Linux)
```